ccd-definition/AuthorisationCaseField/AuthorisationCaseField.json

```diff
@@ -907,7 +907,6 @@
       {
         "UserRoles": [
           "caseworker-civil-admin",
```

Comment:
If this is going with the prod-nonprod release technique, we should make all changes to existing fields in the nonprod file and take out the base one to the prod file (so the base file no longer contains it).